qa/SidechainTestFramework/account/requirements.txt

```diff
@@ -0,0 +1,4 @@
+eth_abi
+pycryptodome
+rlp
+eth_utils
```

Comment:
For me the names were "eth-abi" and "eth-utils"

Reply:
I needed to specifically install pycryptodome, the others were not enough for me. These are the packages I'm using, so while rlp is probably always going to be a dependency for eth_abi and eth_util I'd still rather keep it as an extra mention.